index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
- [ ] As asked previously please add a descriptive Pull Request description `the file added on the first project` is not descriptive enough
- [ ] I just realized that the readme file is on the `main` branch but please remember that the readme file should be only present in the `feature` branch in your  case ` helloworld_project` branch because this branch is being reviewed